test/foundry-tests/UniswapV2.t.sol

```diff
@@ -33,7 +36,8 @@ abstract contract UniswapV2Test is Test {
             v2Factory: address(FACTORY),
             v3Factory: address(0),
             pairInitCodeHash: bytes32(0x96e8ac4277198ff8b6f785478aa9a39f403cb768dd02cbee326c3e7da348845f),
-            poolInitCodeHash: bytes32(0)
+            poolInitCodeHash: bytes32(0),
+            v3NFTPositionManager: address(V3_POSITION_MANAGER)
```

Comment:
i feel like it can just be address(0) here given this file just tests uniswap v2